packages/core/src/App/Containers/AccountTypesModal/account-types-modal.jsx

```diff
@@ -1,224 +1,24 @@
-import { Icon, Modal, Tabs, ThemedScrollbars } from '@deriv/components';
+import { Icon, Modal, ThemedScrollbars } from '@deriv/components';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { withRouter } from 'react-router';
-import { website_name } from 'App/Constants/app-config';
+import { getDerivComLink, urlFor } from '@deriv/shared/utils/url';
```

Comment:
Please use the `@deriv/shared'` import. This will not work, as the `@deriv/shared/utils` path does not exist anymore. Please do the same for the other `@deriv/shared` imports too.

```suggestion
import { getDerivComLink, urlFor } from '@deriv/shared';
```